_health-care/_js/components/personal-information/VAInformationSection.jsx

```diff
@@ -17,17 +17,29 @@ class VaInformationSection extends React.Component {
           <ErrorableCheckbox
               label="Are you VA Service Connected 50% to 100% Disabled?"
               checked={this.props.data.isVaServiceConnected}
+              required={!false}
```

Comment:
I don't think it's a required field. That wouldn't make sense, what would you do if your answer to this is false? You have no choice but to not check it.